state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -886,14 +905,23 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 					amount,
 					destination_address,
 					egress_id,
-				} => {
-					egress_ids.push(egress_id);
-					transfer_params.push(TransferAssetParams {
-						asset,
-						amount,
-						to: destination_address,
-					});
-				},
+				} =>
+					if amount > MinimumEgress::<T, I>::get(asset) {
```

Comment:
"Minimum Egress" sounds as if it stands for the smallest, but still an allowed amount: I'd put a `>=` here.

Reply:
Needs to be `>` because this will be 0 in some cases. If we make this >= 0 we will try egressing 0 amounts. I clarified in the comment on the storage - also made it consistent with the `BITCOIN_DUST_AMOUNT` that we use in the Bitcoin tx construction - we now use strictly greater everywhere, so it all ties nicely together (without any `-1`s all over the place).

Reply:
On second thought, I guess an alternative is to make the minimum 1 for the other chains 🤔 . That seems like a (very slightly) worse approach though